pkg/permissionclaim/permissionclaim_labeler.go

```diff
@@ -100,14 +101,16 @@ func (l *Labeler) LabelsFor(ctx context.Context, cluster logicalcluster.Name, gr
 				continue
 			}
 
-			k, v, err := permissionclaims.ToLabelKeyAndValue(logicalcluster.From(export), export.Name, claim.PermissionClaim)
-			if err != nil {
-				// extremely unlikely to get an error here - it means the json marshaling failed
-				logger.Error(err, "error calculating permission claim label key and value",
-					"claim", claim.String())
-				continue
+			if IsSelected(claim.PermissionClaim, resourceName, resourceNamespace) {
```

Comment:
What would you think about moving the check for ClaimAccepted and matching group+resource into IsSelected? And maybe renaming to Selects? So it would look like:

```go
if !Selects(claim, groupResource, resourceNamespace, resourceName) {
  continue
}
// do labeling
```

Reply:
That makes a lot of sense to me. Will change.